lib/core/templates/empty-content/empty-content.component.stories.ts

```diff
@@ -0,0 +1,113 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Meta, moduleMetadata, Story } from '@storybook/angular';
+import { EmptyContentComponent } from './empty-content.component';
+import { CoreStoryModule } from '../../testing/core.story.module';
+import { TemplateModule } from '../template.module';
+
+export default {
+    component: EmptyContentComponent,
+    title: 'Core/Template/Empty Content',
+    decorators: [
+        moduleMetadata({
+            imports: [CoreStoryModule, TemplateModule]
+        })
+    ],
+    argTypes: {
+        icon: {
+            description: 'Angular Material icon',
+            table: {
+                category: 'Component Inputs',
+                type: {
+                    summary: 'string'
+                },
+                defaultValue: {
+                    summary: 'cake'
+                }
+            }
+        },
+        title: {
+            table: {
+                category: 'Component Inputs',
+                type: {
+                    summary: 'string'
+                },
+                defaultValue: {
+                    summary: ''
+                }
+            }
+        },
+        subtitle: {
+            table: {
+                category: 'Component Inputs',
+                type: {
+                    summary: 'string'
+                },
+                defaultValue: {
+                    summary: ''
+                }
+            }
+        },
+        lines: {
+            name: 'lines',
+            description: 'Content Projection Text',
+            type: { name: 'object', required: false },
+            defaultValue: [
+                'Items you removed are moved to the Trash',
+                'Empty Trash to permanently delete items'
+            ],
+            table: {
+                category: 'Strories Controls',
+                type: {
+                    summary: 'array'
+                }
+            }
+        }
+    }
+} as Meta;
+
+const template: Story = args => ({
+    props: args
+});
+
+export const defaultStory = template.bind({});
+defaultStory.argTypes = {
+    lines: {
+        control: { disable: true }
+    }
+};
+defaultStory.args = {
+    icon: 'star_rate',
+    title: 'No favourite files or folders',
+    subtitle: 'Favourite items that you want to easily find later'
+};
+defaultStory.storyName = 'Default';
+
+export const multipleLines: Story = args => ({
+    props: {
+        ...args
+    },
+    template: `
+    <adf-empty-content icon="delete" title="Trash is empty">
+        <p class="adf-empty-content__text" *ngFor="let line of ${JSON.stringify(
```

Comment:
do we need that css class here (in the story)?

Reply:
In my opinion, it can stay that way, because that CSS class is presented in [documentation](https://github.com/Alfresco/alfresco-ng2-components/blob/develop/docs/core/components/empty-content.component.md). `adf-empty-content__text` class is also implemented in component CSS file so it is related strictly with EmptyContent.